doc/Doxyfile.in

```diff
@@ -193,8 +193,8 @@ TAB_SIZE               = 4
 # will result in a user-defined paragraph with heading "Side Effects:".
 # You can put \n's in the value part of an alias to insert newlines.
 
-ALIASES                = "assert=\par Assertions:\n" \
-                         "default=\par Default Value:\n"
+ALIASES                = "assert=\par Assertions:" \
```

Comment:
Tese two lines alone fix a lot of the errors Doxygen spits out. I patched this locally but never sent it in because I was eventually gonna batch it with some other Doxygen fluff. Good work!

Reply:
@NotTsunami Do you perhaps know how to get Doxygen to include assertions on its own? Currently, assertions are inserted manually, but with recent fuzzing fixes some of these have changes and I've forgotten which ones. Searching for 'Doxygen extract assertions' gives some results that seem to indicate it is possible, but I can't find out how.

The one thing I did find out is that I probably need to add `"FLAC_ASSERT(x)=assert(x)"` to the `PREDEFINED`, but that alone doesn't seem to be enough.